packages/whale-api-client/package.json

```
@@ -0,0 +1,37 @@
+{
+  "private": false,
```

```suggestion
  "private": true,
```

Set this to private first to prevent CI from publishing this until it's ready for production use.